src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -5173,6 +5171,21 @@ public String toString() {
 	}
 
 	/**
+	 * Transform this {@link Flux} in order to generate a target {@link Flux}.
```

Comment:
"Unlike compose(), the provided function is executed during assembly time. Example use:

Function<Flux<Integer>, Flux<Integer>> applySchedulers = flux -> flux.subscribeOn(Schedulers.io()).publishOn(Schedulers.computation());

source.transform(applyScheduler).map(v -> v * v)...
"